nixos/modules/services/web-servers/keter/bundle.nix

```diff
@@ -0,0 +1,35 @@
+/* This makes a keter bundle as described on the github page:
+   https://github.com/snoyberg/keter#bundling-your-app-for-keter
+
+   Note that we're not actually putting in the app into the bundle,
+   we already can use the nix store for copying, so we just
+   symlink to the app.
+*/
+{ domain,
+  exePkg,
+  executable, gnutar, writeTextFile, lib, stdenv, ... }:
+
+let
+  str.stanzas = [{
+    # we just use nix as an absolute path so we're not bundling any binaries
+    type = "webapp";
+    exec = "${exePkg}/bin/${executable}";
+    host = "${domain}";
+  }];
+  configFile = writeTextFile {
+    name = "keter.yml";
+    text = (lib.generators.toYAML { } str);
+  };
+
+in stdenv.mkDerivation {
+  name = lib.strings.sanitizeDerivationName "keter-bundle";
```

Comment:
It looks like a trivial/safe name to me, why sanitize?

Reply:
sorry, this must have leaked trough all my copy paste editing. I shall remove the sanitization